ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -838,7 +845,7 @@ export default class ConfirmTransactionBase extends Component {
         txData.metadata.note = noteText;
       }
 
-      if (isSmartTransactionsEnabled) {
+      if (isSmartTransaction) {
         txData.origin += '#smartTransaction';
       }
```

Comment:
Can we now get rid of `isSmartTransaction` related code from here ?

Reply:
This is still needed by MMI, I've just used an existing variable with a more explicit name.